app/controllers/users_controller.rb

```diff
@@ -0,0 +1,54 @@
+class UsersController < ApplicationController
+  before_action :require_user, only: [:dashboard]
+
+  def dashboard
+    @user = User.find(session[:user_id])
+  end
+
+  def loginpage
```

Comment:
Can someone walk me through why we need this method *and* the `#new` method as well?